packages/obonode/obojobo-chunks-question/viewer-component.test.js

```diff
@@ -160,150 +201,124 @@ describe('Question', () => {
 		jest.resetAllMocks()
 	})
 
-	test('Question component', () => {
-		const moduleData = {
-			questionState: 'mockQuestionState',
-			navState: {
-				context: 'mockContext'
-			},
-			focusState: 'mockFocus'
-		}
-		const model = OboModel.create(questionJSON)
+	test('Question in practice not answered', () => {
+		expect(createComponent('default', 'practice', null, false).toJSON()).toMatchSnapshot()
+	})
 
-		// Question with no score data
-		QuestionUtil.getScoreForModel.mockReturnValueOnce(null)
-		QuestionUtil.getViewState.mockReturnValueOnce('mockViewState')
+	test('Question in practice answered incorrectly', () => {
+		expect(createComponent('default', 'practice', 0, false).toJSON()).toMatchSnapshot()
+	})
 
-		const component = renderer.create(<Question model={model} moduleData={moduleData} />)
-		const tree = component.toJSON()
+	test('Question in practice answered correctly', () => {
+		expect(createComponent('default', 'practice', 100, false).toJSON()).toMatchSnapshot()
+	})
 
-		expect(tree).toMatchSnapshot()
+	test('Survey question in practice not answered', () => {
+		expect(createComponent('survey', 'practice', null, false).toJSON()).toMatchSnapshot()
```

Comment:
I'd really prefer to see a second expect that looks for the value expected specifically like `QuestionUtil.getScoreForModel.mockReturnValueOnce(100)`